### PR TITLE
Create ocxl_afp3.c

### PR DESCRIPTION
**Pull request by fieldsaj**
Adding latest AFP code to Samples, per JT's request.

### CODE REVIEW

File: samples/AFP/ocxl_afp3.c [hunk 1]
```diff
@@ -0,0 +1,599 @@
+/*
+ *(C) Copyright 2017 International Business Machines
+ */
```
Comment (ajdlinux) [inline samples/AFP/ocxl_afp3.c:15]: Please add a licence header that matches the rest of the project

File: samples/AFP/ocxl_afp3.c [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+#include <time.h>
+
+#include "libocxl.h"
+#define DEVICE "IBM,AFP3"
+
+//  CACHELINE_BYTES 64
```
Comment (ajdlinux) [inline samples/AFP/ocxl_afp3.c]: can we get rid of this? (and the other commented out lines)

**Comment by ajdlinux** [comment]
Thanks for posting this!

We need to at least fix the licence header and the remaining commented out code.

I think @philippe56 is working on adding more AFU testing code so you might need to coordinate with him.

**Comment by fieldsaj** [comment]
License header and commented code have been corrected. Thanks for the reminder on those.

**Comment by deece** [comment]
I'd like to keep the samples barebones, so unless this illustrated libocxl API usage that doesn't occur in the memcpy or lpc samples, I think this would be better off passed on to @philippe56 as another test application.

**Comment by philippe56** [comment]
@fieldsaj Thank you for the afp test program.
I have integrated it (along with a memcpy test program) in the new afutests subdirectory.
See PR https://github.com/OpenCAPI/libocxl/pull/17

**Comment by deece** [comment]
Brought in as part of afutests, thanks

**Pull request closed without merging**